src/main/resources/android/res-map.txt

```diff
@@ -0,0 +1,12779 @@
+# The base resources can be found at https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/core/res/res/values/
+# Parsed from android.jar (Android-33) from resources.arsc
```

Comment:
android.jar contains all the base resources used by every app, each app can also have its own resources (we handle this already when creating the primary resource)